packages/notifier/src/publish-kit.js

```diff
@@ -180,12 +200,221 @@ export const makePublishKit = () => {
       advanceCurrent(true, finalValue);
     },
     fail: reason => {
-      const rejection = HandledPromise.reject(reason);
-      // Suppress unhandled rejection error.
-      rejection.catch(() => {});
-      advanceCurrent(true, undefined, rejection);
+      advanceCurrent(true, undefined, makeQuietRejection(reason));
     },
   });
   return harden({ publisher, subscriber });
 };
 harden(makePublishKit);
```

Comment:
Not urgent, but I would like to understand why we still need notifier.js . AFAICT, subscribers subsume notifiers, as we intended. Where does this fail?

Reply:
Likewise for subscriber.js . Can we at least deprecate these and start migrating all uses to publish-kit?

Reply:
There was an issue with implementing notifiers on top of publish kit: https://github.com/Agoric/agoric-sdk/pull/5435#issuecomment-1158874885

But FWIW, I do consider both notifier and subscriber to be deprecated in favor of publish-kit.